portfolio/src/test/java/com/google/sps/models/RequestDataTest.java

```diff
@@ -0,0 +1,122 @@
+package com.google.sps.models;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.Before;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.MockitoAnnotations;
+import org.mockito.Mockito;
+import javax.servlet.http.HttpServletRequest;
+import java.util.NoSuchElementException;
+import java.io.IOException;
+import com.google.sps.exceptions.InvalidMultipartRequest;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Collection;
+import java.util.List;
+import java.util.Arrays;
+import javax.servlet.http.Part;
+import org.apache.commons.io.IOUtils;
+import javax.servlet.ServletException;
+import com.google.sps.exceptions.InvalidMultipartRequest;
+
+/**
+ * Unit tests for the Course class.
+ */
+ @RunWith(JUnit4.class)
+public final class RequestDataTest {
+  private RequestData requestData;
+
+  private static final String param1 = "username";
+  private static final String body1 = "user1";
+  private static final String param2 = "content";
+  private static final String body2 = "hello, this is a comment!";
+
+  @Before
+  public void setUp() {
+    MockitoAnnotations.initMocks(this);
+    testConstructor();
+  }
+
+  @Test
+  public void testConstructor() {
```

Comment:
if this is all supposed to be setup, move it directly into your setUp() method.  Also it is not a constructor.